named_arrays/_matrices/tests/test_matrices.py

```diff
@@ -31,7 +31,24 @@ def test_is_consistent(self, array: na.AbstractMatrixArray):
         assert not array_2.is_consistent
 
     def test_is_square(self, array: na.AbstractMatrixArray):
-        assert array.is_square
+        assert array.is_square == (len(array.row_prototype.entries) == len(array.column_prototype.entries))
+
+    def all_vectors(array: na.AbstractVectorArray):
```

Comment:
If this is supposed to be a test, it needs to start or end with the slug `test`

Reply:
oops, don't need that.